index.html

```diff
@@ -411,385 +410,88 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
-<dfn>decodedBodySize</dfn> attribute MUST return as follows:</p>
-<ol>
-<li>The size, in octets, received from a <a data-cite=
-"FETCH#http-network-or-cache-fetch">HTTP-network-or-cache</a>
-fetch, of the <a data-cite="RFC7230#section-3.3">message body</a>
-[[RFC7230]], after removing any applied <a data-cite=
-"RFC7231#section-3.1.2.1">content-codings</a> [[RFC7231]], if the
-resource passes the <a>timing allow check</a> algorithm.</li>
-<li>The size, in octets, of the <a data-cite=
-"RFC7230#section-3.3">payload</a> after removing any applied
-<a data-cite="RFC7231#section-3.1.2.1">content-codings</a>, if the
-resource is retrieved from <a data-cite=
-"HTML#relevant-application-cache">relevant application caches</a>
-or from local resources.</li>
-<li>zero, otherwise.</li>
+<p data-dfn-for="PerformanceResourceTiming">When <dfn>toJSON</dfn>
+is called, run [[WEBIDL]]'s <a data-cite=
+"WEBIDL#default-tojson-operation">default toJSON operation</a>.</p>
+<p data-dfn-for="PerformanceResourceTiming">
+<dfn>initiatorType</dfn> getter steps would be to return the
+<a data-for="PerformanceResourceTiming">initiator type</a> for <a>this</a>.
+<p data-dfn-for="PerformanceResourceTiming">The <dfn>workerStart</dfn> getter
+steps are to <a>convert fetch timestamp</a> for <a>this</a>'
+<a data-for="PerformanceResourceTiming">timing info</a>'s
+[=fetch timing info/final service worker start time=] and the <a>relevant global object</a> for
+<a>this</a>. See [=/HTTP fetch=] for more info.
+<p data-dfn-for="PerformanceResourceTiming">The <dfn>redirectStart</dfn> getter steps are to
+<a>convert fetch timestamp</a> for <a>this</a>' <a data-for="PerformanceResourceTiming">timing
+info</a>'s [=fetch timing info/redirect start time=] and the <a>relevant global object</a>
+for <a>this</a>. See [=/HTTP-redirect fetch=] for more info.
+<p data-dfn-for="PerformanceResourceTiming">The <dfn>redirectEnd</dfn> getter steps are to
+<a>convert fetch timestamp</a> for <a>this</a>' <a data-for="PerformanceResourceTiming">timing
+info</a>'s [=fetch timing info/redirect end time=] and the <a>relevant global object</a> for
+<a>this</a>. See [=/HTTP-redirect fetch=] for more info.
+<p data-dfn-for="PerformanceResourceTiming">The <dfn>fetchStart</dfn> getter steps are to
+<a>convert fetch timestamp</a> for <a>this</a>' <a data-for="PerformanceResourceTiming">timing
+info</a>'s [=fetch timing info/post-redirect start time=] and the <a>relevant global object</a>
+for <a>this</a>. See [=/HTTP fetch=] for more info.
+<p data-for="PerformanceResourceTiming">The <dfn>domainLookupStart</dfn> getter steps are to
+<a>convert fetch timestamp</a> for <a>this</a>' <a data-for="PerformanceResourceTiming">timing
+info</a>'s [=fetch timing info/final connection timing info=]'s
+[=connection timing info/domain lookup start time=] and the <a>relevant global object</a> for
+<a>this</a>.
+See <a data-cite="FETCH#concept-recording-connection-timing-info">Recording connection timing
```

Comment:
I guess this link will only work after landing the FETCH part? Not currently linking to anything

Reply:
fixed